src/app/schedule-view/component.ts

```diff
@@ -142,6 +144,25 @@ export class ScheduleViewComponent implements OnInit, OnDestroy {
     return this.schedules[this.scheduleIndex].statusText;
   }
 
+  public showExport () : void {
+      this.isExporting = true;
+  }
+
+  public downloadImage () : void {
+      var node = document.getElementById('schedule-container');
+
+      domtoimage.toPng(node,{bgcolor:"white",quality:1.0})
+        .then(function (dataUrl) {
+            var link = document.createElement('a');
+            link.download = 'mySchedules.png';
+            link.href = dataUrl;
+            link.click();
+        })
+        .catch(function(error) {
+          console.error('oops, something went wrong!', error);
```

Comment:
Could we make this error more specific?

Reply:
@Bad-Science  Actually specific error description is in error parameter